params/mavproxy.param.default

```diff
@@ -2,10 +2,10 @@
 --load-module='GPSInput,DepthOutput'
 --source-system=200
 --cmd="set heartbeat 0"
+--cmd="param set SERIAL0_PROTOCOL 2"
 --out udpin:localhost:9000
 --out udpout:localhost:9002
 --out udpin:0.0.0.0:14660
 --out udpbcast:192.168.2.255:14550
 --mav20
---aircraft telemetry
```

Comment:
Mavproxy can still fail if we reboot and the log file fills the entire space, we should save the telemetry under /tmp file to be sure that we'll not have such problem in the future:
` --logfile=/tmp/telemetry.log`